### PR TITLE
Support meetingRegistrantGet operation

### PR DESCRIPTION
**Pull request by ammancilla**
https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/meetingRegistrantGet

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -45,8 +44,7 @@ GEM
     method_source (1.0.0)
     mini_mime (1.1.5)
     minitest (5.18.0)
-    multi_xml (0.7.1)
-      bigdecimal (~> 3.1)
```
Comment (ammancilla) [inline Gemfile.lock:49]: Updated `httparty` which depends on a version of `multi_xml` that does not exist anymore.

> Your bundle is locked to multi_xml (0.7.1) from rubygems repository
https://rubygems.org/ or installed locally, but that version can no longer be
found in that source. That means the author of multi_xml (0.7.1) has removed it.

Comment (kyleboe) [reply Gemfile.lock:49]: Awesome! Thanks for fixing that!

**Review by kyleboe — APPROVED** [review]
Thanks for the contribution! I'll get this merged and a new version published here soon!

**Comment by ammancilla** [comment]
> Thanks for the contribution! I'll get this merged and a new version published here soon!

Thank you!

**Comment by klappradla** [comment]
> Thanks for the contribution! I'll get this merged and a new version published here soon!

Hey @kyleboe 👋 Any updates on a release with this change in? Would be super happy about that 🙏

**Pull request merged**